src/object/coin.cpp

```diff
@@ -335,4 +344,12 @@ HeavyCoin::after_editor_set()
   MovingSprite::after_editor_set();
 }
 
+// Call on_flip from grandparent class HeavyCoin to avoid flipping of
+// gravity-affected object HeavyCoin
+void
+HeavyCoin::on_flip(float height)
+{
+  MovingSprite::on_flip(height);
+}
```

Comment:
Detail, but many IDEs interpret comments *before* a function to be documentation; it would be preferable to have the comment inside the function's body.

Reply:
I only just noticed I wrote that the grandparent of HeavyCoin is HeavyCoin :D
I'll fix that too (probably this evening)